src/main/webapp/index.html

```diff
@@ -49,8 +49,8 @@ <h2>MentorMatch</h2>
 </div>
 	
 <!--
-<div id="footer-wrapper">
-	<div id="footer" class="container">
+<div id="footer-wrapper"
```

Comment:
You need to have closing brackets, so the original code should be correct. Please revert.